docs/CONTRIBUTING.md

```diff
@@ -69,6 +69,38 @@ systems as they can before sending a pull request.
 *The test suite is run automatically by CI servers when you send a pull
 request.*
 
+Diffing Binaries
+----------------
+
+Binary files are tagged as "binary" in the `.gitattributes` file, but also have
+a `diff=hex` tag, which allows you to see hexdump-style diffs for binaries,
+if you add the following to either your global or repository local git config:
+
+```ini
+[diff "hex"]
+  textconv = hexdump
+  binary = true
+```
```

Comment:
Is this section necessary, given that you show the relevant `git config` commands below?